docs/apex_development_setup.md

```diff
@@ -94,6 +94,42 @@ When the run completes successfully, you should see a new directory named `myape
 
 The build should create the application package file `myapexapp/target/myapexapp-1.0-SNAPSHOT.apa`. This application package can then be used to launch example application via **dtCli**, or other visual management tools.  When running, this application will generate a stream of random numbers and print them out, each prefixed by the string `hello world:`.
 
+Running Unit Tests
+----
+To run unit tests on Linux or OSX, simply run the usual maven command, for example: `mvn test`.
+
+On Windows, an additional file, `winutils.exe`, is required; download it from
+<https://github.com/srccodes/hadoop-common-2.2.0-bin/archive/master.zip>
+and unpack the archive to, say, `C:\hadoop`; this file should be present under
+`hadoop-common-2.2.0-bin-master\bin` within it.
+
```

Comment:
Yes, I tried downloading master.zip, unpacking and setting the environment variable locally within the IDE as well as on the console on Windows 10.

Describing how to set the environment variable globally is different in different versions of Windows and covering all variants would introduce too much verbiage into the document. There are tons of web pages that show how to do it, I just picked one.